src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -718,6 +734,7 @@ protected function redirectAfterSave($isNewRecord)
             $url = $controller->getRequest()->getURL();
             $noActionURL = $controller->removeAction($url);
             $controller->getRequest()->addHeader('X-Pjax', 'Content');
+            $controller->getRequest()->addHeader('X-Status', rawurlencode(_t(__CLASS__ . '.SAVEDUP', 'Saved successfully') ?? ''));
```

Comment:
Can you please explain how I can trigger this one?

Reply:
Actually, I couldn't find how to reproduce this. Probably, I remove this for now.